Source/WebCore/dom/Document.h

```diff
@@ -333,6 +333,10 @@ enum class DimensionsCheck : uint8_t {
     All = 1 << 2, // FIXME: This is probably meant to be Width | Height instead.
 };
 
+enum class LayoutOptions : uint8_t {
+    RunPostLayoutTasksSynchronously = 1 << 0,
```

Comment:
Should the "IgnorePendingStylesheets" behavior also be part of the options?

Reply:
I like that idea! It makes the OptionSet a lot more useful in my opinion.
In my update I still have updateLayoutIgnorePendingStylesheets in order to not have to update the many callsites, however I have no problem updating them here or in a followup. But also let's see if it goes green....